pyaedt/emit_core/__init__.py

```diff
@@ -0,0 +1 @@
+from pyaedt.emit_core.results import Results
```

Comment:
I think it will be best not to bring these classes up into the package. For now at least, let's try keeping this __init__ file empty. I think that will be the least confusing. We might want to pull things into the pyaedt.emit package so the user can do pyaedt.emit.xyz, but I don't think I see a need to do it here in emit_core, or in results/__init__.py either. Does that make sense?